### DISCUSSION TITLE
[SUGGESTION]: Ability to display README before the code files

### DISCUSSION THREAD

**Discussion by zaaakher** [community/community#128922] [Repositories]
### Select Topic Area

Product Feedback

### Body

Hey 👋,

It would be nice if we can have a repository option to display the README content before the list of files. I realized I always scroll down to the README first before going through the code whenever I discover a new repository. I have a feeling many github users have a similar habit 😅.

If I enable the README-first option, it would look something like this 👇:

![image](https://github.com/community/community/assets/46135573/7cd6329b-10c5-49a9-8b1d-673721c883e6)

**Comment by zaaakher** [comment]
And perhaps there could be a collapse button to collapse the readme if people don't want to scroll through a long README

**Comment by SaviPandey** [comment]
This approach ensures that the README, which often serves as the entry point for understanding a project, is prominently displayed. It would certainly align with common user behavior and improve accessibility to key information.

I hope GitHub considers implementing such a feature in the future!
